### ISSUE TITLE
[BUG] Campaign : browser parameter

### ISSUE THREAD

**Issue by Galand-V** [cerberustesting/cerberus-source#1742]
Hello, 

The **browser parameter** is not mandatory for campaign creation (useless for API testcases)

When I try to launch an API campaign with no browser the run page is freeze with the loader. 
It should be an issue with the addtoexecutionqueue servlet.

**Comment by vertigo17** [comment]
I could not reproduce.
This is what I did :
1/ I defined a campaign with no robot and browser defined.
2/ I get to RunTest page and select Campaign then click on Load
3/ I forced on robot : -- Custom Config -- as Cerberus default to MyRobot (left all fields empty including browser)
![image](https://user-images.githubusercontent.com/5342048/40805485-b32e5682-651e-11e8-8444-e74eb2d86710.png)

4/ I clicked on Run Campaign and campaign got submitted with no issue.
![image](https://user-images.githubusercontent.com/5342048/40805510-c486c6ee-651e-11e8-8326-2ac1fc9bf645.png)
